packages/now-build-utils/src/prerender.ts

```diff
@@ -7,21 +7,26 @@ interface PrerenderOptions {
   expiration: number;
   lambda: Lambda;
   fallback: FileBlob | FileFsRef | FileRef;
-  group: number;
+  group?: number;
 }
 
 export class Prerender {
   public type: 'Prerender';
   public expiration: number;
   public lambda: Lambda;
   public fallback: FileBlob | FileFsRef | FileRef;
-  public group: number;
+  public group?: number;
 
   constructor({ expiration, lambda, fallback, group }: PrerenderOptions) {
     this.type = 'Prerender';
     this.expiration = expiration;
     this.lambda = lambda;
     this.fallback = fallback;
+    
+    if (group && group <= 0 || !Number.isInteger(group)) {
```

Comment:
`group &&` will short circuit for zero or undefined, which is why `Number.isInteger` warns about the type.

This is the correct logic that passes type checking:

```suggestion
    if (
      typeof group !== 'undefined' &&
      (group <= 0 || !Number.isInteger(group))
    ) {
```